texi2rst.py

```diff
@@ -484,6 +484,22 @@ def postvisit_element(self, element, parent):
     return tree
 
 
+def fixup_text_variables(tree):
+    class TextVariableFixer(NoopVisitor):
+        REPLACEMENTS = (('$$VERSION_PACKAGE$$', '|package_version|'),
+                        ('$$version-GCC$$', '|gcc_version|',),
+                        ('$$BUGURL$$', '|bugurl|'))
```

Comment:
Is the `|var|` syntax from .rst/sphinx, or something you've invented?

Reply:
There are defined in `baseconf.py`, it uses standard `::replace` directive:
https://github.com/davidmalcolm/texi2rst/blob/46d828911d9c59fe96e6cc2105014c9003050e8e/scripts/templates/baseconf.py#L47-L51